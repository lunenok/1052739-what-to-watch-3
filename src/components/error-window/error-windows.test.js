import ErrorWindow from "./error-window.jsx";
import React from "react";
import renderer from "react-test-renderer";

it(`Should add review page render correctly`, () => {
  const defaultMessage = `Выполните вход`;

  const tree = renderer
    .create(
        // <Provider store={store}>
        <ErrorWindow message={defaultMessage}/>
        // </Provider>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
