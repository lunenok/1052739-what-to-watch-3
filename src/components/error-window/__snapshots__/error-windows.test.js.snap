// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should add review page render correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "fontSize": "30px",
      "left": 0,
      "margin": "0 auto",
      "position": "absolute",
      "right": 0,
      "textAlign": "center",
      "top": 0,
      "zIndex": 100,
    }
  }
>
  Выполните вход
</div>
`;
